numba_dpex/experimental/launcher.py

```diff
@@ -297,8 +402,11 @@ def cleanup(
 
 
 @intrinsic(target="cpu")
```

Comment:
I now realize that the target keyword on `intrinsic` is a no-op. I am not sure if it is a feature or a bug in Numba.